src/mscorlib/shared/System/StringComparer.cs

```diff
@@ -170,6 +171,7 @@ public int GetHashCode(object obj)
     }
 
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     internal sealed class CultureAwareComparer : StringComparer
```

Comment:
> Actually it doesn't mean we cannot serialize them it just means that we won't be able to deserialize across .NET Core and .NET Framework

Right, that's what I meant.  That's the context of this TypeForwardedFrom change.

> is that a goal for all the types we plan to support for binary serialization?

Yes, for any types we ship as [Serializable] in .NET Core 2.0.